typescript/web/src/components/auth/sign-in.tsx

```diff
@@ -6,8 +6,8 @@ import {
 } from "@chakra-ui/react";
 import NextLink from "next/link";
 import React from "react";
-import { Logo } from "../logo";
 import { TextLink } from "../text-link";
+import { UkrainianLogo } from "../website/ukrainian-logo";
```

Comment:
Can we avoid referencing the specific `UkranianLogo` in every page and just do the switch in the correct `Logo` component instead?